symfony/apps/orangehrm/templates/vue.html.twig

```diff
@@ -0,0 +1,13 @@
+{% extends 'base.html.twig' %}
+
+{% block body %}
+    <noscript>
+        <strong>
+            We're sorry but orangehrm doesn't work properly without JavaScript enabled. Please enable it to continue.
+        </strong>
+    </noscript>
+
+    <div id="app">
+        <oxd-layout><{{ tag }}></{{ tag }}></oxd-layout>
```

Comment:
is <tag> make sense ? or <page-content-component> 
and how you pass props here ?